src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -513,8 +513,7 @@ public function iShouldBeNotifiedThatThisValueShouldNotBeBlank()
     }
 
     /**
-     * @Then I should be notified that a percentage discount value must be at least 0%
-     * @Then I should be notified that the maximum value of a percentage discount is 100%
```

Comment:
I would be for leaving these two steps as they are still valid and someone may have used them in the end application